services/director-v2/tests/unit/test_modules_dynamic_sidecar_client_api_base.py

```diff
@@ -121,7 +121,7 @@ async def raises_request_error(self) -> Response:
         connections_message = caplog_info_level.messages[-1]
         assert (
             connections_message
-            == "Pool status @ 'POOL TIMEOUT': requests=[], connections=[]"
+            == "Pool status @ 'POOL TIMEOUT': requests(0)=[], connections(0)=[]"
```

Comment:
Here is (again) the inconvenient of testing messages :-) 
(again) some alternative:
- mock logger function
- use constants for messages
- check keywords in message instead of entire message